src/DestroyCommand.php

```diff
@@ -21,9 +21,8 @@ protected function configure()
     /**
      * Execute the command.
      *
-     * @param  \Symfony\Component\Console\Input\InputInterface  $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
-     * @return void
```

Comment:
Running it from package root yields no changes, which seems inconsistent
due to the MakeCommand.php having namespace on a different line than other
commands, but it must be something in the .php_cs allowing it so I digress.

On Sun, Jul 5, 2015 at 2:22 PM, Graham Campbell <notifications@github.com>
wrote:

> In src/DestroyCommand.php
> <https://github.com/laravel/homestead/pull/226#discussion_r33898003>:
>
> > @@ -21,9 +21,8 @@ protected function configure()
> >      /**
> >       * Execute the command.
> >       *
> > -     * @param  \Symfony\Component\Console\Input\InputInterface  $input
> > -     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
> > -     * @return void
>
> Passing any parameters to php-cs-fixer makes it avoid using the config
> file. Don't pass src.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/homestead/pull/226/files#r33898003>.
>



-- 
- Joe Ferguson
MemphisTechnology.org
MemphisPHP.org
HACKMemphis.com